hack/e2e-test.sh

```diff
@@ -42,18 +42,26 @@ function startup {
         if [ ! -d "$ARTIFACTS" ]; then
             mkdir -p "$ARTIFACTS"
         fi
-	cluster_create "$KIND_CLUSTER_NAME"  "$SOURCE_DIR/$KIND_CLUSTER_FILE" 
+	      cluster_create "$KIND_CLUSTER_NAME"  "$SOURCE_DIR/$KIND_CLUSTER_FILE"
     fi
 }
 
 function kind_load {
-    if [ "$CREATE_KIND_CLUSTER" == 'true' ]
-    then
-        docker pull "$E2E_TEST_IMAGE"
-	cluster_kind_load "$KIND_CLUSTER_NAME"
+    prepare_docker_images
```

Comment:
Nice, so we can save installing CRDs specifically to e2e test suite. 
I would like to extract this to a preparatory PR for the ease of reviewing.

Reply:
https://github.com/kubernetes-sigs/kueue/pull/3606